### ISSUE TITLE
nesting has trouble parsing multi-prop options

### ISSUE THREAD

**Issue by FortinFred** [i18next/i18next#1403]
I have the following configuration:

```
    nestingKey: '$t(CONTEXT_KEY, { "context": "A", "count": 2 })'
    CONTEXT_KEY_A: 'GOOD',
    CONTEXT_KEY_A_plural: 'GOODS',
```

I'm expecting `GOODS`.

Getting: `"A", "count": 2 }`

**Comment by FortinFred** [comment]
Also tried:
```
'$t(CONTEXT_KEY, { "context": "A", "count": "2" })'
```

Getting: `"A", "count": "2" })'`

**Comment by jamuhl** [comment]
not sure what you try todo...but this is not the way it works. https://www.i18next.com/translation-function/nesting#passing-options-to-nestings

**Comment by FortinFred** [comment]
Well the full example was something with defaultValue but I got stuck when adding more than one option into the context object:

```
    nestingKey: '$t(CONTEXT_KEY, { "context": "{{dynamicContext}}", "default": "Something Unknown" })'
    CONTEXT_KEY_A: 'Got Type A',
    CONTEXT_KEY_B: 'Got Type B',

```

**Comment by jamuhl** [comment]
defaultValue gets deleted from options: https://github.com/i18next/i18next/blob/master/src/Interpolator.js#L182 <- reason

**Comment by FortinFred** [comment]
So there's no situation where the $t would receive options with multiple props?

If so, I still think there's an issue.

**Comment by jamuhl** [comment]
Seems like a regression...works in v19.1.0 but fails now: https://jsfiddle.net/jamuhl/4gdf1298/

**Comment by jamuhl** [comment]
Will need to find a new solution which works for https://github.com/i18next/i18next/issues/1395 and obviously multiple options in the options string that gets parsed.

**Comment by jamuhl** [comment]
Should be fixed in i18next@19.3.2

https://jsfiddle.net/jamuhl/oLh06gs3/
